### DISCUSSION TITLE
SQLAlchemy declarative_base support

### DISCUSSION THREAD

**Discussion by AmitArie** [python-gino/gino#805] [Q&A]
Hello!
I'm using SQLAlchemy & Alembic in order to manage my MySQL DB.
In order to use python classes, I use the `declarative_base` of SQLAlchemy:

```
from sqlalchemy.ext.declarative import declarative_base
Base = declarative_base()

class Organization(Base):
    __tablename__ = "organization"
    __table_args__ = {"extend_existing": True}

    id = sqlalchemy.Column(sqlalchemy.Integer, primary_key=True)
    name = sqlalchemy.Column(sqlalchemy.String(512), nullable=False)
```

There is a way to use this model in gino instead of migrating it to inherit  from db.model?
Thanks!

**Comment by fantix** [comment]
What is your use case of using both `declarative_base` and `db.model` please?
